source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service.py

```diff
@@ -106,25 +106,28 @@ def _call_service(self, params, headers={}):
             )
             raise RidesharingServiceError(str(e))
 
-    def request_journeys_with_feed_publisher(self, from_coord, to_coord, period_extremity, instance, limit=None):
+    def request_journeys_with_feed_publisher(
+        self, from_coord, to_coord, period_extremity, instance_params, limit=None
+    ):
         """
         This function shouldn't be overwritten!
 
         :return: a list(mandatory) contains solutions and a feed_publisher
         """
         try:
-            journeys = self._request_journeys(from_coord, to_coord, period_extremity, instance, limit)
+            journeys = self._request_journeys(from_coord, to_coord, period_extremity, instance_params, limit)
             feed_publisher = self._get_feed_publisher()
 
             self.record_call('ok')
 
             return journeys, feed_publisher
         except RidesharingServiceError as e:
             self.record_call('failure', **e.get_params())
+            self.logger.exception('')
```

Comment:
It's necessary ?

Reply:
yes, there are some exceptions that are not logged... (PS: `self.logger.exception` logs the stack of exception, even though the message is empty)